pkg/controller/kogitoapp/resource/kogito_imagestream_test.go

```diff
@@ -89,14 +89,6 @@ func TestKogitoImageStreamGeneration(t *testing.T) {
 			assert.Equal(t, imgv1.TagReferencePolicy{Type: imgv1.LocalTagReferencePolicy}, item.Spec.Tags[0].ReferencePolicy)
 			assert.Equal(t, "DockerImage", item.Spec.Tags[0].From.Kind)
 			assert.Equal(t, fmt.Sprintf("quay.io/kiegroup/%s:%s", item.Name, version.Version), item.Spec.Tags[0].From.Name)
-
-		case KogitoDataIndexImage:
```

Comment:
shouldn't we keep this?

Reply:
No, the image stream is being created by the Data Index controller.